shared-module/displayio/Display.c

```diff
@@ -111,6 +111,8 @@ void common_hal_displayio_display_construct(displayio_display_obj_t* self,
     // Always set the backlight type in case we're reusing memory.
     self->backlight_inout.base.type = &mp_type_NoneType;
     if (backlight_pin != NULL && common_hal_mcu_pin_is_free(backlight_pin)) {
+        // Avoid PWM types and functions when the module isn't enabled
+        #if (CIRCUITPY_PULSIO)
```

Comment:
```suggestion
        #if (CIRCUITPY_PULSEIO)
```

Please test on a non-STM board as well with a display that PWMs the backlight. The #ifs here have a typo.

Reply:
Whoops, good point and good catch. I'll make a note to include an out-of-port control when testing shared module.